package.json

```diff
@@ -34,7 +34,8 @@
     "compile-all": "npm run compile-ts && npm run compile-compass && npm run compile-exec",
     "prerelease": "npm run compile-cli",
     "release": "node scripts/release.js",
-    "publish-npm": "lerna publish --force-publish",
+    "publish-npm": "node scripts/publish-npm.js",
+    "publish-npm-lerna": "lerna publish --force-publish",
```

Comment:
I would remove this script from package.json, because it might be confusing to figure out which one to run if you're just reading the `scripts` section of package.json.

Reply:
which one `publish-npm` or `publish-npm-lerna`?

Reply:
line 38, so `publish-npm-lerna`.